src/components/breadcrumb/BreadcrumbLink/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,23 @@
+import classnames from 'classnames'
+import React from 'react'
+import {
+  CustomLinkProps,
+  DefaultLinkProps,
+  isCustomProps,
+  Link,
+} from '../../Link/Link'
+
+export function BreadcrumbLink(props: DefaultLinkProps): React.ReactElement
+export function BreadcrumbLink<T>(props: CustomLinkProps<T>): React.ReactElement
+export function BreadcrumbLink<FCProps = DefaultLinkProps>(
+  props: DefaultLinkProps | CustomLinkProps<FCProps>
+): React.ReactElement {
+  const { className } = props
+  const classes = classnames(className, 'usa-breadcrumb__link')
+
+  if (isCustomProps(props)) {
+    return <Link<FCProps> {...props} className={classes} variant="unstyled" />
+  }
+
+  return <Link {...props} className={classes} variant="unstyled" />
+}
```

Comment:
I've now got `BreadcrumbLink` wrapping `Link`. I did not want to re-implement any logic from the `Link` component, to prevent duplicated maintenance in the event the Link implementation were to change. 

I was hoping there would be a way to wrap the Link component's signature completely in more of a HOC fashion, and simply inject the `className`. Due to Link's clever overloading involving the optional generic, I had to mirror at least that part here.